### ISSUE TITLE
New feature

### ISSUE THREAD

**Issue by shainodev** [UnamSanctam/UnamWebPanel#150]
Do you ever think of considering to add more features to the webpanel like rats have? run file from url for example or something like that?

**Comment by UnamSanctam** [comment]
No I don't intend for the web panel to be like a RAT, it would be much better to just use a normal RAT for that purpose.

**Comment by shainodev** [comment]
> No I don't intend for the web panel to be like a RAT, it would be much better to just use a normal RAT for that purpose.

what about a stealer?

**Comment by UnamSanctam** [comment]
If I make a stealer it would be a separate program (I have one that's almost done but haven't finished it), but if I finish it I would most likely integrate it into the web panel as well (a new tab).

**Comment by shainodev** [comment]
> If I make a stealer it would be a separate program (I have one that's almost done but haven't finished it), but if I finish it I would most likely integrate it into the web panel as well (a new tab), haven't finished it since making http(s) calls in C isn't very fun.

oh, good job. yeah i believe that, from my point of view coding anything makes my brain really hurt and i can never make anything perfect so i just look for other people to do it

**Comment by shainodev** [comment]
yo estimatetly how long will it take

**Comment by UnamSanctam** [comment]
It's not my priority at the moment.

**Comment by DogusCyk** [comment]
ah are you making a stealer as a separate program?

**Comment by UnamSanctam** [comment]
Yes a stealer isn't a miner and shouldn't really be connected to a miner so it would be a separate program.
